README.md

```diff
@@ -1003,6 +1003,43 @@ const MyResponsiveDiv = glamorous.div({
 ```
 </details>
 
+In addition to all this, you can reference a `glamorous` component inside the
+styles of other `glamorous` components (this works because they each implement
+the `toString` method):
+
+<details>
+<summary>Contextual Overrides</summary>
+
+This API is inspired by `styled-components`.
+
+```javascript
+const Link = glamorous.a({
+  color: 'inherit',
+  textDecoration: 'none',
+  '&:hover, &:active': {
+    textDecoration: 'underline',
+  }
+})
+
+const P = glamorous.p({
+  [`& ${Link}`]: {
```

Comment:
It really is unfortunate... We _could_ make a glamor plugin though! That would work! If there's a selector that starts with `.class-name` we'll insert an `&` in front of it. That would work :+1:

Give me a bit of time to update for that.